ThunderCloud/ButtonListItem.swift

```diff
@@ -54,8 +54,6 @@ open class ButtonListItem: ListItem {
     /// - Parameter dictionary: A Storm object dictionary.
 	public required init(dictionary: [AnyHashable : Any]) {
 		super.init(dictionary: dictionary)
-		
-		configure(with: dictionary, languageController: StormLanguageController.shared)
```

Comment:
Ahhhh, because it subclasses from `ListItem` now which does all this configure shizzle too!